linkerd/protocol/http/src/main/scala/io/buoyant/linkerd/protocol/HttpConfig.scala

```diff
@@ -244,7 +244,7 @@ case class HttpConfig(
     .maybeWith(httpAccessLogAppend.map(AccessLogger.param.Append.apply))
     .maybeWith(httpAccessLogRotateCount.map(AccessLogger.param.RotateCount.apply))
     .maybeWith(loggerParam)
-    .maybeWith(combinedIdentifier)
+    .maybeWith(combinedIdentifier(routerParams))
```

Comment:
is this infinitely recursive?  should we pass all of the params EXCEPT the identifier param to the identifier?

Reply:
Ya silly mistake - fixed and ran test - all passed